helm-chart/kuberay-operator/README.md

```diff
@@ -16,6 +16,11 @@ version.BuildInfo{Version:"v3.6.2", GitCommit:"ee407bdf364942bcb8e8c665f82e15aa2
 
 ## Installing the Chart
 
+To avoid duplicate CRD definitions in this repo, we reuse CRD config in `ray-operator`:
+```console
+$ kubectl apply -k "kuberay/ray-operator/config/crd"
```

Comment:
```suggestion
$ kubectl apply -k "../../ray-operator/config/crd"
```
Because the other commands in the readme assume the current dir is _helm-chart/kuberay-operator_.